docs/rfc/rfc-1.adoc

```diff
@@ -253,12 +253,26 @@ wallet and closes.
 
 === Wallet Lifecycle
 
-Wallets are periodically created, where the period length is a governable
-parameter. To create a new wallet, a group of 100 operators is selected from
-the pool of available operators (some operators may be selected twice if there
-are not enough) using a process called sortition. The probabiliy that a
-particular operator is chosen is based on their stake weight, which in turn is
-based on the number of `T` tokens they have invested in the staking contract.
+Governable Parameters:
+
+- `wallet_dust_threshold` the smallest amount of btc a wallet can hold before we
+  attempt to close the wallet and transfer the funds to a randomly selected
+  wallet
+- `wallet_creation_period` how frequently we attempt to create new wallets
+- `wallet_max_btc` the amount of btc a wallet can hold before we trigger the
+  early creation of a new wallet
+- `max_wallet_age` the oldest we allow a wallet to become before we transfer the funds
+  to a randomly selected wallet
+
+A new wallet is created when either enough time has passed as defined in
+`wallet_creation_period` *AND* the wallet contains at least
+`wallet_dust_threshold` btc *OR* the wallet currently contains more btc than
+`wallet_max_btc`, whichever happens first. To create a new wallet, a group of
+100 operators is selected from the pool of available operators (some operators
+may be selected twice if there are not enough) using a process called
```

Comment:
> some operators may be selected twice if there are not enough

It is not entirely true. Some operators can be selected twice or more even if there are enough operators in the pool. The probability depends on the stake weight and nothing else.

Reply:
updated in 75a7457